support-frontend/assets/helpers/redux/checkout/csrf/reducer.ts

```diff
@@ -0,0 +1,10 @@
+import { createSlice } from '@reduxjs/toolkit';
+import { initialCsrfState } from './state';
+
+export const csrfSlice = createSlice({
+	name: 'csrf',
+	initialState: initialCsrfState,
+	reducers: {},
+});
```

Comment:
So there are no reducers? i.e we never update this state?

Reply:
Yep, we found the unused action/reducer in the original PR that introduced it, and the comment in the PR notes it never gets used: https://github.com/guardian/support-frontend/pull/126/files#r127286535